.github/workflows/elixir.yml

```diff
@@ -6,40 +6,39 @@ on:
 jobs:
   build:
     name: Build and test
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
     env:
       MIX_ENV: test
       ELIXIR_ENV: test
     strategy:
       matrix:
-        otp: ['22.3.4', '23.2.7', '24.1']
-        elixir: ['1.11.4', '1.12.3']
+        otp: ["22.3.4", "23.2.7", "24.1", "26.1.2"]
```

Comment:
@Kwilliams-dev303 can you update this to only include:

```
        otp: ["24.1", "26.1.2"]
        elixir: ["1.15.7"]
```

This release would drop support for all lower Elixir versions than 1.15.7

Reply:
@sb8244 sure thing

Reply:
And otp should actually be `24.3.4` and not `24.1`. The new builders require specific patch numbers

Reply:
@sb8244 gotcha 👍 pushing change

Reply:
Booms tests are running now!! Fingers crossed 🤞

Reply:
Ayy! Looks like we got one failure though. It passes locally for me.